website/docs/guides/version-6-upgrade.html.markdown

```diff
@@ -271,6 +272,18 @@ terraform import aws_api_gateway_stage.prod <rest_api_id>/<stage_name>
 * `compute_environment_name` has been renamed to `name`.
 * `compute_environment_name_prefix` has been renamed to `name_prefix`.
 
+## resource/aws_bedrock_model_invocation_logging_configuration
+
+The following arguments are now list nested blocks instead of single nested blocks.
+
+- `logging_config`
+- `logging_config.cloudwatch_config`
+- `logging_config.cloudwatch_config.large_data_delivery_s3_config`
+- `logging_config.s3_config`
+
+When referencing these arguments, the indicies must now be included in the attribute address.
```

Comment:
```suggestion
When referencing these arguments, the indices must now be included in the attribute address.
```

Reply:
🤦 - thanks. I made this typo a few other places, so will fix them all and rebase.